http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/monitor/MonitorAtBoot.scala

```diff
@@ -331,22 +332,24 @@ class MonitorAtBoot[F[_]](publisherQueue: Queue[F, LeoPubsubMessage],
               F.raiseError(MonitorAtBootException("Azure runtime shouldn't have non Azure runtime config", traceId))
           }
           implicit0(appContext: Ask[F, AppContext]) <- F.pure(Ask.const(AppContext(traceId, now)))
-          relayNamespaceOpt <- wsmDao.getRelayNamespace(wid, azureRuntimeConfig.region, leoAuth)
           storageContainerOpt <- wsmDao.getWorkspaceStorageContainer(wid, leoAuth)
           storageContainer <- F.fromOption(
             storageContainerOpt,
             BadRequestException(s"Workspace ${wid} doesn't have storage container provisioned appropriately",
                                 Some(traceId)
             )
           )
-          rns <- F.fromOption(relayNamespaceOpt,
-                              MonitorAtBootException(s"no relay namespace found for ${wid}", traceId)
-          )
+          workspaceDescOpt <- wsmDao.getWorkspace(wid, leoAuth)
+          workspaceDesc <- F.fromOption(workspaceDescOpt, WorkspaceNotFoundException(wid, traceId))
+
+
+          // Get the Landing Zone Resources for the app for Azure
+          landingZoneResourcesOpt <- wsmDao.getLandingZoneResources(workspaceDesc.spendProfile, leoAuth)
```

Comment:
I'm not familiar with this code path, and am not sure the leoAuth token has permission to request landingZone resources (the above wsm calls are both for wsm managed resources). TODO to check on this-

Reply:
It probably doesn't! :) You could maybe use https://github.com/DataBiosphere/leonardo/blob/develop/http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/dao/SamDAO.scala#L105 to call the LZ API

Reply:
This reminds me that I don't think we touched `MonitorAtBoot` for the app stuff -- this code tries to replay messages if Back Leo restarts while it is monitoring something.

Reply:
For the Sam API, I'm still skeptical since we had the previous issue with apps where the pet access token wasn't available. I'm a little confused about how leoAuth works in general, I'll reach out to chat about it-